packages/api/src/external/carequality/document/get-non-existent-doc-refs.ts

```diff
@@ -16,19 +16,41 @@ const parallelS3Queries = 10;
 export async function getNonExistentDocRefs(
   documents: DocumentReferenceWithMetriportId[],
   patientId: string,
-  cxId: string
+  cxId: string,
+  // TEMP #2349 - Remove console.log
+  log?: typeof console.log
 ): Promise<DocumentReferenceWithMetriportId[]> {
   const [{ existingDocRefs, nonExistingDocRefs }, fhirDocRefs] = await Promise.all([
     checkDocRefsExistInS3(documents, patientId, cxId),
     getDocumentsFromFHIR({ cxId, patientId }),
   ]);
 
+  // TEMP #2349 - Remove console.log
+  if (log) {
+    log("existingDocRefs", `${existingDocRefs.length}`, JSON.stringify(existingDocRefs));
```

Comment:
It should be fine b/c all params are strings, but usually we have it all in one string to avoid having multi-line logs that break Cloudwatch's logs